src/screens/authentication/Login/index.js

```diff
@@ -33,14 +34,21 @@ const Login = () => {
   return (
     <Auth>
       <Auth.Navigation goTo='SignUp'>Registrarme</Auth.Navigation>
-      <Auth.Title>Iniciá sesión</Auth.Title>
+      <Auth.Logo />
+      <Auth.Title
+        emailSent={emailSent}
+      >Iniciá sesión
+      </Auth.Title>
```

Comment:
This is not a good way to customize a component conditionally. What if you need to change this same component in different screens for different reasons? Will you add a new prop for each reason that needs to change the style?

The correct and generic way to do this is passing a `style` prop from the outside, and combine the styles inside the component. The default style always comes first and the outside style last:
```javascript
const defaultStyle = // create the default style

function CustomizableComponent ({ children, style }) {
  return (
    <View style={[defaultStyle, style]}>
      {children}
    </View>
  )
}
```